build-with-ubp/action.yml

```diff
@@ -0,0 +1,65 @@
+name: 'Build with UBP'
+description: 'Creates a build with UBP'
+
+inputs:
+  files:
+    description: 'The files you want to build with space delimited'
+    required: true
+  override:
+    description: 'Your override json for the job'
+    required: true
+  platform:
+    description: 'The target platform'
+    required: true
+  unityVersion:
+    description: 'The version of unity for building'
+    required: true
+  buildWithUnityCore:
+    description: 'Toggle on to use Unity Core execute method'
+    default: false
+    required: false
+    type: boolean
+
+runs:
+  using: composite
+  steps:
+    - name: 'Run UBP'
+      env: 
+        UNITY_COMMAND: '/Applications/Unity/Hub/Editor/${{ inputs.unityVersion }}/Unity.app/Contents/MacOS/Unity -projectPath ${{ github.workspace }} -batchmode -logfile -'
```

Comment:
There's an extra `-` at the end of this variable's value.

Reply:
intended
Added '-' for the -logfile make it print to console

Reply:
We may also want to make the path at the front (`/Applications/Unity/Hub/Editor`) configurable since people may use this on existing machines that aren't setup like our build machines. We can default it to our path though.